docs/user/ai_model_serving.md

```diff
@@ -0,0 +1,517 @@
+# AI Model Serving on MicroShift
+
+AI Model Serving on MicroShift is a single-model serving platform for AI models.
+It includes limited subset of Red Hat OpenShift AI (RHOAI):
+[KServe in Raw Deployment Mode](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.17/html/serving_models/serving-large-models_serving-large-models#raw_deployment_mode)
+and the `ServingRuntimes` object type that is namespace-specific.
+Now you can train your models in the cloud and serve them on the edge.
+
+## Definitions
+
+- [ServingRuntime](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.17/html/serving_models/serving-large-models_serving-large-models#servingruntime)
+  - For more information about Serving Runtimes refer to [upstream Kserve documentation](https://kserve.github.io/website/latest/modelserving/servingruntimes/)
+- [InferenceService](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.17/html/serving_models/serving-large-models_serving-large-models#inferenceservice)
+
+## Supported model-serving runtimes
+
+Currently MicroShift supports following model-serving runtimes:
+- OpenVINO Model Server
+- vLLM ServingRuntime for KServe
+
+Following runtimes are included but not supported at the time:
```

Comment:
What does it mean? We cannot separate those out from the image? 
The question that requires explaining is: why do we include those if we do not support them?

Reply:
I'm not sure we need to emphasize what we're testing or not in the docs - it may change quickly.
If models are there, let's just mention them? If we specifically support (not test) selected models, that's important distinction.

Reply:
These are tested and supported by RHOAI, so just like we don't retest every possible feature of other optional components, I think we can say these are supported even without tests (especially for "upstream" doc)